meilisearch/client.py

```diff
@@ -672,3 +673,21 @@ def _valid_uuid(uuid: str) -> bool:
         )
         match = uuid4hex.match(uuid)
         return bool(match)
+
+    def create_snapshot(self) -> TaskInfo:
+        """Trigger the creation of a Meilisearch snapshot.
+
+        Returns
+        -------
+        Snapshot:
+            Information about the snapshot.
+            https://www.meilisearch.com/docs/reference/api/snapshots#create-a-snapshot
+
+        Raises
+        ------
+        MeilisearchApiError
+            An error containing details about why Meilisearch can't process your request. Meilisearch error codes are described here: https://www.meilisearch.com/docs/reference/errors/error_codes#meilisearch-errors
+        """
+        task = self.http.post(self.config.paths.snapshots)
```

Comment:
The `snapshot` path needs to be added to the `Paths` in `config.py`

Reply:
added it